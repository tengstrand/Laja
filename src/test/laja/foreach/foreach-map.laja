#write "target/test-output/foreach-map.out"
  #foreach (x in {a:"a1" b:"b2" c:"c3"})
value = {x}
  #end
#end
