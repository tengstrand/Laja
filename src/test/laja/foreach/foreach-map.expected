value = {key=a, value=a1}
value = {key=b, value=b2}
value = {key=c, value=c3}
